_posts/2020-10-16-changelog-133.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #133"
+date: 2020-10-19 12:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5189 -->
+* Support configuring workspace features in `Cargo.toml`.
+  Now it is possible to enable or disable any specific workspace [feature](https://doc.rust-lang.org/cargo/reference/features.html) in the `Cargo.toml` file, and our name resolution and other code insight will take them into account.
+  **TODO: more info**
+  **TODO: gif**
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5444 -->
+* Resolve stdlib and Cargo source code hyperlinks in backtrace (by [@Kobzol])
+  {% include gif-img.html path="/assets/posts/changelog-133/5444" w="600px" %}
+
+## Fixes
+
+* UI improvements:
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/6172 -->
+  * Use recommended guidelines for settings UI
+  
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/6179 -->
+  * Get rid of scrollbar inside project creation dialog in CLion
+  
+* Parser improvements:
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/6177 -->
+  * Properly parse `tuple.0.0` as tuple indexing
+
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/6202 -->
+  * Recover TOML parser on errors in arrays
+  
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/6143 -->
+  * Properly parse special macro calls with fully qualified names like `std::include`
+  
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6163 -->
+* Grazie plugin integration improvements:
+  * Fix incorrect proofreading annotations on raw and binary string literals
+  * Support `String literals`, `Comments` and `Documentation` in `Preferences | Editor | Proofreading | Grammar`
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6184 -->
+* Fix ``Make `self` mutable`` quick-fix of `E0594` compiler error (by [@ondralukes])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6196 -->
+* Do not remove required parentheses by `Remove dbg!` intention (by [@t-kameyama])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6176 -->
+* Show test results in the gutter via red and green icons to differ passed and failed tests
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6178 -->
+* Use public visibility when creating function in other crate by `Create function` intention
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6258 -->
+* Update `HashMap` and `HashSet` debugger pretty-printers to Rust 1.47
```

Comment:
It's not clear that we fix proper rendering of `HashMap` and `HashSet` in our UI debugger